src/components/sections/KeysDataSection.tsx

```diff
@@ -28,6 +41,90 @@ const KeysDataSection = ({ query }) => {
         }
     )
 
+    const { searchTerm, setSearchTerm, filteredData } = useSearchFilter(
+        data?.results
+    )
+
+    const numberOfKeysInNamespace = data?.results?.length
+
+    const handleCreate = async ({ key }) => {
+        await engine.mutate(
+            {
+                type: 'create',
+                resource: `${store}/${currentNamespace}/${key}`,
+                data: () => ({}),
+            },
+            {
+                onComplete() {
+                    setOpenCreateModal(false)
+                    showSuccess(
+                        i18n.t("Key '{{key}}' added successfully", {
+                            key,
+                        })
+                    )
```

Comment:
Success alert bars are not necessary for minor actions ([see UI docs](https://ui.dhis2.nu/components/alertbar#when-not-to-use)), so I suggest removing this. The user can see that it was successful because it appears in the list.